package-lock.json

```diff
@@ -13160,7 +13160,41 @@
 				"@wordpress/browserslist-config": "file:packages/browserslist-config",
 				"@wordpress/element": "file:packages/element",
 				"@wordpress/warning": "file:packages/warning",
+				"browserslist": "^4.16.6",
 				"core-js": "^3.6.4"
+			},
+			"dependencies": {
+				"browserslist": {
```

Comment:
Can we bump `browserslist` version in other places? Here:
https://github.com/WordPress/gutenberg/blob/b5f6955742e784f11828e1dacba4361713135963/package.json#L154

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/31898/commits/9898955bf5e2bc8995970da736983f05f9966920 with these steps:

1. `npm install browserslist@latest --save-dev`
2. Removed browserslist from the package again
3. `npm i`
4. `lerna add browserslist packages/babel-preset-default`

Not sure if steps 2-4 were necessary, probably not.

Reply:
I guess this section should have been removed with the update though? Is it okay to remove it manually or did I do something wrong?

Reply:
Yes, updating mannually `dependencies` key and its value should do the trick. You only need to run `npm i` afterward to ensure it's all good.

Reply:
Thanks, looks good now. I also updated `caniuse-lite` with `npx browserslist@latest --update-db` which did remove the extra `caniuse-lite` dependency.

```
❯ npx browserslist@latest --update-db
npx: Installierte 6 in 1.414s
Latest version:     1.0.30001228
Installed versions: 1.0.30001214, 1.0.30001228
Removing old caniuse-lite from lock file
....

❯ npx browserslist@latest --update-db
npx: Installierte 6 in 1.316s
Latest version:     1.0.30001228
Installed version:  1.0.30001228
caniuse-lite is up to date
```